.vscode/launch.json

```diff
@@ -13,7 +13,7 @@
             },
             "args": "${input:funcArgs} --script-root ${input:scriptRootArg}",
             "cwd": "${workspaceFolder}/src/Cli/func",
-            "console": "internalConsole",
+            "console": "integratedTerminal",
```

Comment:
Is this change unrelated to flaky test fix?

Reply:
No, just an enhacement for the repro in general - didn't feel like it was worth its own pr